avssync/avssync.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 type AvsSync struct {
-	AvsReader       avsregistry.AvsRegistryReader
-	AvsWriter       avsregistry.AvsRegistryWriter
+	AvsReader       avsregistry.ChainReader
+	AvsWriter       avsregistry.ChainWriter
```

Comment:
what's this change exactly? Haven't touched the sdk in a while so not sure. Is this the change that fixes the "no slasher founder" issue?

Reply:
yeah so changing to the new avs chainreader and chainwriter fixes the issue because they're no longer checking for the slasher which no longer exists on holesky

Can see the diff: https://github.com/Layr-Labs/eigensdk-go/pull/342/files
![Screenshot 2025-01-09 at 11 37 14 AM](https://github.com/user-attachments/assets/8a2fbca9-ee7e-402c-b614-c57393446546)

Reply:
Oh this is interface renaming 
We had bad naming - avsRegistry.AvsRegistryChainReader. We replaced it with avsRegistry.ChainReader

Reply:
And what afk said - this version of sdk removes slasher contract binding generation so fixed the issue.

Reply:
I see, thanks!